eclipse-theia/theia
# Update the SCM Plugin API from the latest vscode

Signed-off-by: Igor Vinokur <ivinokur@redhat.com>



#### What it does

Update the SCM Plugin API methods from the latest vscoe.

related CQ: http://dev.eclipse.org/ipzilla/show_bug.cgi?id=23027
fixes https://github.com/eclipse-theia/theia/issues/7953
#### How to test

1. Remove `"@theia/git": "^1.10.0"` from `examples/browser/package.json` file.
2. Download [vscode-git-1.49.3](https://open-vsx.org/api/vscode/git/1.49.3/file/vscode.git-1.49.3.vsix) and copy it to the `plugins` folder.
3. Start Theia and  open a project cloned from GitHub e.g. https://github.com/vinokurig/environment-variable.git
4. Open browser logs console and make some git operations with the help of the scm view.
See: no repeating `root ERROR Child node 'file-change-tree-root' does not belong to this 'file-change-tree-root' tree.` errors.
#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
